tools/docker/docker-compose-mac.yml

```diff
@@ -12,7 +12,7 @@ x-environment:
   - EDA_PODMAN_SOCKET_URL="unix:///run/podman/podman.sock"
   - EDA_CONTROLLER_URL=${EDA_CONTROLLER_URL:?Please specify EDA_CONTROLLER_URL env}
   - EDA_CONTROLLER_SSL_VERIFY=${EDA_CONTROLLER_SSL_VERIFY:-yes}
-  - EDA_PODMAN_EXTRA_ARGS='@json {"network_mode":"host"}'
+
```

Comment:
@bzwei Without the network_mode host I couldn't get the activation to work on the M1 Mac.

Reply:
I tested this PR with the newer podman machine images and it seems to work. So this is good.